finetune/base_models/textcnn/model.py

```diff
@@ -0,0 +1,20 @@
+from finetune.base_models import SourceModel
+from finetune.base_models.gpt.encoder import GPTEncoder
+from finetune.base_models.textcnn.featurizer import textcnn_featurizer
+
+class TextCNNModel(SourceModel):
+
+    encoder = GPTEncoder
+    featurizer = textcnn_featurizer
+    kernel_sizes = [2, 3, 4]
```

Comment:
Worth running some experiments on enso to find some good defaults for this.

Reply:
May also make sense to include unigram features.